test/runtests.jl

```diff
@@ -64,9 +72,9 @@ cd(dirname(@__FILE__)) do
         error("Some tests exited with errors.")
     end
 
```

Comment:
The `rmproc` was actually added in https://github.com/JuliaLang/julia/blob/066c2459023fe7658494116c9eb38cde218c3348/test/runtests.jl  and the `addproc` was made `@unix_only` in https://github.com/JuliaLang/julia/blob/9d26ae1fec3a18b2969701b0661fe8f676e50df6/test/runtests.jl, presumably because it didn't used to work on windows...